egs2/bur_openslr80/asr1/run.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+lid=false # whether to use language id as additional label
+
+train_set="bur_train"
+train_dev="bur_dev"
+test_set="bur_test"
+
+asr_config=conf/train_asr_hubert_transformer_adam_specaug.yaml
+inference_config=conf/decode_asr.yaml
+lm_config=conf/train_lm.yaml 
+ngpu=1
+
+./asr.sh \
+    --stage 1 \
+    --stop_stage 100 \
+    --ngpu ${ngpu} \
+    --nj 80 \
+    --inference_nj 256 \
+    --inference_asr_model valid.acc.best.pth \
```

Comment:
Is there specific reason that you add the argument?

Reply:
This has been left unchanged from the reference recipe (bn_openslr53) so I assumed it was required?

Reply:
It is not required, and usually we would use the averaged model instead of the single best. But given that you are using that for your current reported result, it would be fine.